test/test_scanner.py

```diff
@@ -820,6 +820,23 @@ def _file_test(self, url, filename, package, version):
                     "CVE-2019-3836",
                 ],
             ),
+            (
+                "test-GNUTLS-CLI-2.1.6.out",
```

Comment:
As with the bzip one, please rename this to be all lowercase to match the other tests for gnutls.